test/unit/util.test.ts

```diff
@@ -65,3 +67,45 @@ test('Get full URL', async(t) => {
             'https://wikimedia.org/api/rest_v1/media/math/render/svg/34cbb1e27dae0c04fc794a91f2aa001aca7054c1',
             'Full Url when base and url both strtas with http/s');
 })
+
+test('Custom flavour path', async(t) => {
+
+    // checks in current working directory.
+    let createStream = fs.createWriteStream(pathParser.resolve(process.cwd(), 'testCustomFlavour.js'));
+
+    t.equal(getCustomFlavorPath('testCustomFlavour.js'), pathParser.resolve(process.cwd(), 'testCustomFlavour.js'),
+        'Custom flavour in working directory.');
+
+    // checks when current and working directory have file with same name(preference to working directory).
+    createStream = fs.createWriteStream(pathParser.resolve(process.cwd(), 'extensions/testCustomFlavour.js'));
+
+    t.equal(getCustomFlavorPath('testCustomFlavour'), pathParser.resolve(process.cwd(), 'testCustomFlavour.js'),
+        'When file with same name exist in extensions directory.');
```

Comment:
etc... You have a module file which already exists, use it instead of creating one only for the test.

Reply:
@kelson42 this is to test if correct module is picked when a file with same name exist in both current working directory and extension directory I shouldn't have used current working directory part here but I guess the file is still needed.

Reply:
ok